src/main/java/htsjdk/tribble/example/CountRecords.java

```diff
@@ -188,7 +188,7 @@ public static Index createAndWriteNewIndex(File featureFile, File indexFile, Fea
      */
     public static FeatureCodec getFeatureCodec(File featureFile) {
         // quickly determine the codec type
-        //if (featureFile.getName().endsWith(".vcf") || featureFile.getName().endsWith(".VCF") )
+        //if (featureFile.getName().endsWith(IOUtil.VCF_FILE_EXTENSION) || featureFile.getName().endsWith(".VCF") )
```

Comment:
This change can be removed. By the way, this class can be mark as `@Deprecated` and removed in a following release, no? I think that no one look at example code in HTSJDK, or if so a new repository or a clear package for it might be better (although the Wiki can be used for such kind of examples....

Reply:
not in this PR...